src/main/java/entity/Message.java

```diff
@@ -134,20 +106,12 @@ public long getSenderId () {
         return senderId;
     }
 
-    public String getContentType () {
-        return contentType;
-    }
-
     public String getTextContent () {
         return textContent;
     }
 
-    public long getContentId () {
-        return contentId;
-    }
-
-    public String getLinkToBlob () {
-        return linkToBlob;
+    public Long getAttachmentId () {
```

Comment:
IIUC, this can be null.
Consider returning `OptionalLong` instead.
Let's avoid using null since it is very error prone.

Reply:
Done